setup.py

```diff
@@ -26,12 +26,12 @@ def _resolve_prefix(prefix, type):
     # issue: https://github.com/vcstools/wstool/issues/81
     osx_system_prefix = '/System/Library/Frameworks/Python.framework/Versions'
     if type == 'man':
-        if prefix == '/usr':
+        if prefix == '':
             return '/usr/share'
         if sys.prefix.startswith(osx_system_prefix):
             return '/usr/local/share'
     elif type == 'bash_comp':
-        if prefix == '/usr':
+        if prefix == '':
```

Comment:
I'm not sure quite what the desired behavior is with the prefixes on these.  Honestly, I would think the bash completion script install location and man page install location shouldn't be affected by the install prefix, or should have their own options for install location.  I'd be tempted to remove the conditions completely (just checking for os x and otherwise returning `'/'`)